src/TrackPanel.cpp

```diff
@@ -1270,7 +1270,61 @@ struct VRulersAndChannels final : TrackPanelGroup {
    wxCoord mLeftOffset;
 };
 
-// n channels with vertical rulers, alternating with n - 1 resizers;
+//Simply fills area using specified brush and outlines borders
+class EmptyPanelRect final : public CommonTrackPanelCell
+{
+   int mFillBrushName;
+public:
+   explicit EmptyPanelRect(int fillBrushName)
+      : mFillBrushName(fillBrushName)
+   {
+   }
+
+   ~EmptyPanelRect() { }
+
+   void Draw(TrackPanelDrawingContext& context, const wxRect& rect, unsigned iPass)
+   {
+      if (iPass == TrackArtist::PassBackground)
+      {
+         context.dc.SetPen(*wxTRANSPARENT_PEN);
+         AColor::UseThemeColour(&context.dc, mFillBrushName);
+         context.dc.DrawRectangle(rect);
+         wxRect bevel(rect.x, rect.y, rect.width - 1, rect.height - 1);
+         AColor::BevelTrackInfo(context.dc, true, bevel, false);
+      }
+   }
+
+   std::shared_ptr<Track> DoFindTrack() override
+   {
+       return {};
```

Comment:
... and then this might return non-null.  It may be slightly advantageous to do that.  Certain click-drag-release actions query for the track the mouse is over during the drag, which might not be the same as at the click point.  This information can be found when the mouse is over a vertical ruler or over a resizing area.  It might as well be available too for this small rectangle.

Reply:
What for we do need it now?

Reply:
For instance, if you click in one track to select, then drag the mouse into another track, then the other track also becomes selected.  This happens not only if the mouse drags into the waveform area, but even if it drags into the vertical ruler or control area.  It should happen also if the mouse drags into the new empty cell.  So the empty cell should report which track it is associated with.

This may not be a very important case to fix, that anyone will notice.  However, it is very easily implemented, and makes behavior a bit more consistent, so I say, do it.